app.py

```diff
@@ -33,20 +56,18 @@ def create_app(testing=False) -> Flask:
     flask_app.jinja_env.trim_blocks = True
     flask_app.jinja_env.lstrip_blocks = True
 
-    if testing:
-        flask_app.config.from_object(
-            "config.environments.development.DevelopmentConfig"
+    if (os.environ.get("FLASK_ENV") == "development") | testing:
+        flask_app.config.from_object("config.environments.development.DevelopmentConfig"
         )
+        from config.environments.development import DevelopmentConfig
+        if os.environ.get("PRETTY_PRINT"): DevelopmentConfig.pretty_print()
```

Comment:
🔧 This conditional is included in the fsd_tech library.

Reply:
💯 conditional removed! :)